website/docs/reference/resource-properties/freshness.md

```diff
@@ -32,8 +32,8 @@ sources:
           [count](#count): <positive_integer>
           [period](#period): minute | hour | day
         [filter](#filter): <boolean_sql_expression>
-    [loaded_at_field](#loaded_at_field): <column_name_or_expression>
-    [loaded_at_query](#loaded_at_query) <sql_expression> # v1.10 or higher. Should not be used if loaded_at_field is defined
+      [loaded_at_field](#loaded_at_field): <column_name_or_expression> # changed to config in v1.10
+      [loaded_at_query](#loaded_at_query) <sql_expression> # v1.10 or higher. Should not be used if loaded_at_field is defined
 
     tables:
```

Comment:
Freshness settings under both source: and table: may conflict when you define freshness in both places:
* The table-level config.freshness overrides the source-level config.
* Should we call this out in comments to avoid confusion?